packages/web/.storybook/preview-head.html

```diff
@@ -374,4 +359,169 @@
+    const tableHead = [...table.querySelectorAll('th')];
+    tableHead.forEach(th => {
+      th.setAttribute('lang', lang);
+      if (lang == 'en') {
+        const thText = th.innerText;
+        if (thText == 'Name') {
+          th.innerText = 'Property';
+        } else if (thText == 'Description') {
+          th.innerText = 'Data type';
+        } else if (thText == 'Default') {
+          th.innerText = 'Default value';
+        } else if (thText == 'Control') {
+          const resetBtn = th.querySelector('button');
+          th.innerText = 'Values';
+          resetBtn.setAttribute('style', 'float: right;');
+          th.append(resetBtn);
+        }
+      } else {
+        const thText = th.innerText;
+        if (thText == 'Name') {
+          th.innerText = 'Propriété';
+        } else if (thText == 'Description') {
+          th.innerText = 'Type de donnée';
+        } else if (thText == 'Default') {
+          th.innerText = 'Valeur par défaut';
+        } else if (thText == 'Control') {
+          const resetBtn = th.querySelector('button');
+          th.innerText = 'Valeurs';
+          resetBtn.setAttribute('style', 'float: right;');
+          resetBtn.setAttribute('title', 'Réinitialiser les contrôles');
+          th.append(resetBtn);
+        }
+      }
+    });
+
+    // Table Rows
+    const tableRows = [...table.querySelectorAll('tbody > tr')];
+    tableRows.forEach(tr => {
+      // Expand collapse rows
+      if (tr.hasAttribute('class')) {
+        tr.removeAttribute('title');
+        const cellText = tr.children[0].children[1].innerText.toLowerCase().split(' | ');
+        tr.children[1].innerText = lang === 'en' ? 
+          cellText[0].charAt(0).toUpperCase() + cellText[0].slice(1) : 
+          cellText[1].replace(' items', '').charAt(0).toUpperCase() + cellText[1].slice(1);
+        tr.setAttribute('class', '');
+        tr.children[1].setAttribute('colspan', '4');
+        tr.children[1].setAttribute('class', 'sub-table-heading');
+        tr.removeChild(tr.children[0]);
+      }
+
+      // Loop through table cells
+      const propertyId = `${tr.children[0].innerText.replace('*', '')}-property`;
+      for (let x = 0; x < tr.children.length; x++) {
+        // Format first cell
+        if (x == 0) {
+          tr.children[x].setAttribute('id', propertyId);
+          if (tr.children[x].querySelector('[title=Required]') && lang == 'fr') {
+            tr.children[x].querySelector('[title=Required]').setAttribute('lang', 'fr');
+            tr.children[x].querySelector('[title=Required]').setAttribute('title', 'Obligatoire');
+          }
+        }
+        // Format third cell
+        if (tr.children[x].innerText === '-') {
+          if (lang === 'en') {
+            tr.children[x].setAttribute('title', 'No default value');
+          } else {
+            tr.children[x].setAttribute('title', 'Aucune valeur par défaut');
+            tr.children[x].setAttribute('lang', 'fr');
+          }
+        }
```

Comment:
I can see the new "No default value" added as a td title, but I can't find it on the a11y tree nor when I attempt to use voiceover.

Storybook Embed: In the code builder for all components, rows in the Properties table without a default value indicate that there is no default using a hyphen. This may be clear for sighted users but a text alternative should be provided for screen reader and other assistive technology users (e.g. “No default value”).

Reply:
Right it seems the extra logic I had for that isn't here. Will push it up soon